### PR TITLE
[Bug] Fix kibana-url for kibana-upload command

### PR DESCRIPTION
**Pull request by shravaka**
## Issues
https://github.com/elastic/detection-rules/issues/345

## Summary
Bug fix - missing parenthesis cause error if --kibana-url option is provided to the kibana-upload command

### CODE REVIEW

**Review by rw-access — APPROVED** [review]
thanks @shravaka! once you sign the CLA, we'll merge

**Comment by pwlp** [comment]
Thanks!
I already did sign the agreement, waiting for confirmation that everything went well

**Comment by rw-access** [comment]
Looks like you might be using two GItHub accounts? Just make sure you've signed the CLA with the commit  [`fb64ce2`](https://github.com/elastic/detection-rules/commit/fb64ce272394fdb2ad5adbab620bebc4d4ebb640.patch) and have associated that with the right GitHub account.

**Comment by shravaka** [comment]
I will, sorry for that.

**Pull request merged**
